src/components/calcite-combobox-item/calcite-combobox-item.tsx

```diff
@@ -162,12 +188,15 @@ export class CalciteComboboxItem {
   }
 
   render(): VNode {
+    const isSingleSelect = getElementProp(this.el, "selection-mode", "multi") === "single";
```

Comment:
Shouldn't `selectionMode` be a prop on the combobox and nothing on the combobox item?

Reply:
it is exactly what you describe.

Reply:
this is going and getting the selection mode prop set on the parent. `getElementProp` traverses up the dom tree